src/index.js

```diff
@@ -0,0 +1,77 @@
+import 'express-async-errors';
+
+import compression from 'compression';
+import cors from 'cors';
+import express from 'express';
+import helmet from 'helmet';
+import hpp from 'hpp';
+
+import config from './config.js';
```

Comment:
```suggestion
import config from '@/config.js';
```

we use path alias in this project, please fix the impor path.